docs/releases/1.3.rst

```diff
@@ -0,0 +1,23 @@
+==========================================
+Wagtail 1.3 release notes - IN DEVELOPMENT
```

Comment:
@timorieber This will be removed when Wagtail 1.2 final is released. For now there's still a possibility that the 1.2 release will receive bug reports and subsequent fixes.